integration/spark/app/src/test/resources/spark_scripts/spark_jdbc_column.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018-2023 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+
+from pyspark.sql import SparkSession
+
+# Create a Spark session and configure the Spark jars.packages property
+spark = (
+    SparkSession.builder.master("local")
+    .appName("JDBCColumnJob")
+    .config("spark.openlineage.facets.disabled", "[spark_unknown;spark.logicalPlan]")
+    .getOrCreate()
+)
+
+mysql_url = "jdbc:mysql://mysql:3306/test"
+
+# Columns to select from the MySQL table
+input_columns = ["client_name", "client_category", "client_rating"]
+
+# Read only certain columns from ol_clients MySQL table
+ol_clients_df = (
+    spark.read.format("jdbc")
+    .option("url", mysql_url)
+    .option("driver", "com.mysql.cj.jdbc.Driver")
+    .option("user", "user")
+    .option("password", "password")
+    .option("dbtable", "ol_clients")
+    .load()
+    .select(*input_columns)
```

Comment:
How does this test differ from:
https://github.com/OpenLineage/OpenLineage/blob/main/integration/spark/app/src/test/java/io/openlineage/spark/agent/ColumnLineageIntegrationTest.java#L95